dev-tools/reconfigurator-cli/tests/output/cmd-stdout

```diff
@@ -66,3 +66,56 @@ generated inventory collection ..........<REDACTED_UUID>........... from configu
 ID                                   NERRORS TIME_DONE                
 ..........<REDACTED_UUID>........... 0       <REDACTED_TIMESTAMP> 
 
+> 
+
+> save state.json
+saved planning input, collections, and blueprints to "state.json"
+
+> load state.json
+error: changes made to simulated system: run `wipe system` before loading
```

Comment:
This says to run `wipe system` but the next command in the test is just `wipe` - is `system` implied if omitted?

Reply:
lol I copy pasted this from my wip stuff where I changed the command. will fix

Reply:
Updated.